.github/workflows/dependency-review.yml

```diff
@@ -58,3 +58,5 @@ jobs:
           comment-summary-in-pr: ${{ inputs.comment-summary-in-pr }}
           fail-on-severity: high
           config-file: ./coveo-dependency-allowed-licenses/${{ steps.select-config.outputs.result }}
+          base-ref: ${{ github.event.pull_request.base.sha || 'main' }}
```

Comment:
What about repos like `platform-client` that have a `master` branch instead of `main`? Maybe we could define inputs instead which would be more flexible? I started a branch yesterday without knowing you were also working on this. 😅 

https://github.com/coveo/public-actions/commit/4d40b8ac80c7dc23a9320319979390a1b53481f9

Reply:
Oops! I jumped the gun. I prefer your approach, let's use it 👍 
However, after https://github.com/actions/dependency-review-action/pull/766, I think we should remove the default value